pepper-apis/src/main/java/org/broadinstitute/ddp/db/dto/MedicalProviderDto.java

```diff
@@ -50,6 +55,33 @@ public MedicalProviderDto(
         this.street = street;
     }
 
+    private MedicalProviderDto(
+            String userMedicalProviderGuid,
+            long userId,
+            long umbrellaStudyId,
+            InstitutionType institutionType,
+            PostPatchMedicalProviderRequestPayload requestPayload
+    ) {
+        this.userMedicalProviderGuid = userMedicalProviderGuid;
+        this.userId = userId;
+        this.umbrellaStudyId = umbrellaStudyId;
+        this.institutionType = institutionType;
+        this.institutionName = requestPayload.getInstitutionName();
+        this.physicianName = requestPayload.getPhysicianName();
+        this.city = requestPayload.getCity();
+        this.state = requestPayload.getState();
+    }
+
+    public static MedicalProviderDto createUnspecifiedMedicalProvider(
+            String medicalProviderGuid,
+            long userId,
+            long umbrellaStudyId,
+            InstitutionType institutionTypeCode,
+            PostPatchMedicalProviderRequestPayload requestPayload
+    ) {
+        return new MedicalProviderDto(medicalProviderGuid, userId, umbrellaStudyId, institutionTypeCode, requestPayload);
+    }
```

Comment:
Instead of instantiating a `MedicalProviderDto` with multiple NULL params, let's have a static factory method with a meaningful (I hope) name. Please let me know if you come up with a better one.

Reply:
I'm not sure why we're so "protective" of these DTOs... couldn't we just add a new constructor that defaults some of those fields to `null` instead of adding a static factory method?

Reply:
I'd second the ctor approach. Is there a particular reason for not using it?